tests/functional/test_cli.py

```diff
@@ -1960,6 +1962,50 @@ def test_job_info_wrong_status(self):
             text = "".join(lines)
             assert expect in text
 
+    def test_execute_remote_input(self):
+        """
+        Verify that specified inputs are captured for a limited number of 1 item per ``-I`` option.
+        """
```

Comment:
I'm not sure to understand this description when looking at the execute call that uses only 1 `-I` argument. 
Looks like a bad copy-paste from somewhere else?

Reply:
I missed that one, yes, bad copy-paste.